.github/workflows/test.yml

```diff
@@ -45,7 +42,7 @@ jobs:
           check-latest: true
           cache: "sbt"
       - name: Restore cache
```

Comment:
`actions/setup-java@v4` has caching built in so you no longer need this action. Instead it is good to add `setup-sbt` because Github is gradually removing `sbt` from the runner images.

Reply:
good point, @mzuehlke, fixed in 67f0130